### DISCUSSION TITLE
Versioning confusion

### DISCUSSION THREAD

**Discussion by vedeja** [structurizr/cli#163] [General] [answered]
I had missed the change from numerical versioning of cli, lite and, on premise to the date-like versioning. And now I’m a bit confused. When running lite 2024.03.03 the console logs 2.1.1. Is it possible to explain how it all fits together?

**Comment by simonbrowndotje** [comment, marked as answer]
Lite/on-premises/CLI are calver, whereas the Java libraries they depend upon (structurizr-core|client|dsl|etc) are semver. The console/log will show all versions.

**Reply by vedeja** [reply]
Thank you for the clarification
